frame/bounties/src/lib.rs

```diff
@@ -653,12 +786,656 @@ decl_module! {
 					},
 					_ => return Err(Error::<T>::UnexpectedStatus.into()),
 				}
-
+				// TODO :: Have to recheck
+				// Extend expiry for active subbounty
+				if bounty.activesubbounty.len() > 0 {
```

Comment:
sub bounty expiry is always same as parent so no need to modify anything else here

Reply:
Indeed, good one, will remove & rework on it.

Reply:
Rework done, removed the clutters, now ready for review.